src/ApparentHorizons/Strahlkorper.hpp

```diff
@@ -60,6 +66,9 @@ class Strahlkorper {
   SPECTRE_ALWAYS_INLINE const DataVector& coefficients() const noexcept {
     return strahlkorper_coefs_;
   }
+  SPECTRE_ALWAYS_INLINE DataVector& coefficients() noexcept {
```

Comment:
Put the two `coefficients` methods in a `//@{` `//@}` group.

Reply:
Please do this